network.env.dist

```diff
@@ -1,5 +1,6 @@
 # Coinbase Addresses
 # If this node is being used to mine, block rewards (coinbase rewards) will be paid out to the following addresses.
+# Each of these addresses must be unique and map to the correct shard it has been inputted for. 
```

Comment:
Learn something new everyday
https://www.merriam-webster.com/grammar/is-inputted-a-word